src/controllers/shows/tvrecommended.js

```diff
@@ -332,7 +332,7 @@ export default function (view, params) {
     function onInputCommand(e) {
         if (e.detail.command === 'search') {
             e.preventDefault();
-            Dashboard.navigate('search.html?collectionType=tv&parentId=' + params.topParentId);
+            Dashboard.navigate('search.html?collectionType=tvshows&parentId=' + params.topParentId);
```

Comment:
We should pull the value from the SDK to make this future proof.

```js
import { CollectionType } from '@jellyfin/sdk/lib/generated-client/models/collection-type';
```

```suggestion
            Dashboard.navigate(`search.html?collectionType=${CollectionType.Tvshows}&parentId=${params.topParentId}`);
```